extension/test/event-page-data-webtest.js

```diff
@@ -33,6 +33,7 @@ describe('EventPage', function() {
         [{ name: 'entitya1' }, { name: 'entityb2' }],
         result['urla']
       );
+      assret.isOk(false);
```

Comment:
Typo: assret -> assert? This will fail but maybe not for the reason you expected. Perhaps fine either way?

Reply:
:)